src/components/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -106,81 +134,62 @@
         </v-col>
       </v-row>
     </div>
-  </v-card>
+  </div>
 </template>
 
 <script lang="ts">
-// Libraries
 import { Component, Prop, Mixins, Emit } from 'vue-property-decorator'
-
-// Components
+import { Getter } from 'vuex-class'
 import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
-
-// Dialogs
-import { ConfirmRemoveDialog } from '@/components/dialogs'
-
-// Mixins
-import { CommonMixin, EntityFilterMixin } from '@/mixins'
-
-// Interfaces
+import { CommonMixin } from '@/mixins'
 import { OrgPersonIF } from '@/interfaces'
+import { ActionTypes } from '@/enums'
 
 @Component({
   components: {
-    BaseAddress,
-    ConfirmRemoveDialog
+    BaseAddress
   }
 })
-export default class ListPeopleAndRoles extends Mixins(CommonMixin, EntityFilterMixin) {
-  // Props passed in to this component
-  @Prop({ default: () => [] })
-  private personList: Array<OrgPersonIF>
+export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
+  // Enum for template
+  readonly ActionTypes = ActionTypes
```

Comment:
Every time i see the way we do this, i wonder if there is a different way to define these enums on a class based component. Once again, just thinking aloud! haha no action required

Reply:
AFAIK, this is what Vue still requires. But I agree, it would be nice to not need this.